### DISCUSSION TITLE
[Feature] Config option to set default for album display order

### DISCUSSION THREAD

**Discussion by amykyta3** [immich-app/immich#14592] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Unlike the timeline view, I prefer to have photos show up in "Oldest first" for albums.
It is pretty easy to change this setting, but currently the default is "Newest first" so it is easy to forget the change.

Understandably, others may prefer the opposite. Could this be a user config to set the default order for new albums?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8068
